src/Helper/Service/EscaperFactoryTrait.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\View\Helper\Service;
+
+use Laminas\Escaper\Escaper;
+use Laminas\View\Exception\RuntimeException;
+use Psr\Container\ContainerInterface;
+
+use function assert;
+use function gettype;
+use function is_array;
+use function is_string;
+use function sprintf;
+
+trait EscaperFactoryTrait
+{
+    /** @return non-empty-string */
+    private function viewEncoding(ContainerInterface $container): string
+    {
+        $config = $container->has('config')
+            ? $container->get('config')
+            : [];
+
+        assert(is_array($config));
+        $viewConfig = $this->assertArray('view_manager', $config);
```

Comment:
Is `view_manager` traditionally where the `encoding` value is to be found? I can't remember… I think it may have been dropped entirely because I cannot find it in the MVC skeleton anymore…

Reply:
You are right, there is no reference to it anywhere. But it is a _hidden feature_, so someone may use it.

Reply:
Thanks for confirming. This is exactly what should end up in a ConfigProvider for this component such as #121